website/data/api-navigation.js

```diff
@@ -151,7 +151,7 @@ export default [
       'step-down',
       {
         category: 'storage',
-        content: ['raft', 'raftautosnapshots'],
+        content: ['raft', 'raftautosnapshots', 'raftautopilot'],
```

Comment:
If we want them alphabetized in the sidebar, it should be like this.

```suggestion
        content: ['raft', 'raftautopilot', 'raftautosnapshots'],
```